setup.py

```diff
@@ -29,11 +29,11 @@
     long_description=(
         open("README.rst").read() if os.path.exists("README.rst") else ""
     ),
-    url="https://geopandas.dask.org/",
+    url="https://github.com/geopandas/dask-geopandas",
     project_urls={
-        "Documentation": "https://geopandas.dask.org/",
-        "Source": "https://github.com/dask/dask-geopandas/",
-        "Issue Tracker": "https://github.com/dask/dask-geopandas/issues",
+        "Documentation": "https://github.com/geopandas/dask-geopandas",
```

Comment:
We don't yet have a documentation site, so maybe should just leave it out for now ..

Reply:
But we have the basic instructions in ReadMe here. I think we can keep it.